### PR TITLE
What is a sample

### PR DESCRIPTION
**Pull request by honeybadgeralex**
Samples basic info, also some fixed screenshots

### CODE REVIEW

**Comment by honeybadgeralex** [comment]
I'm thinking of renaming all "What is a X?" docs into "X overview"

File: _getting-started/data-sample-overview.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+produces we use data sample.
+
+A data sample can be retrieved from a component or written manually by the
+integrator. In any case, the sample is then given as input to the next
+component. The receiving component may not require all the data from the sample
+or may require it in completely different order. That’s when you configure
+proper mapping so that the receiving component gets input in accordance with its
+standard.
+
+**NOTE:** Samples exist for flow building and testing purposes. However, the data from samples will go into production when you generate a sample. If you want to avoid this, you can use **Generate Stub Sample** function. It retrieves the component's output template and fills it with fake data that you can discard afterwards.
```
Comment (drobiazko) [inline _getting-started/data-sample-overview.md]: > However, the data from samples will go into production when you generate a sample.

I think I know what you are saying here but I would rephrase it because it is not so obvious and also a bit scary. Maybe something like this?

> Retrieving a sample for an action creates new objects in the target system. If you want to avoid ....

File: _getting-started/data-sample-overview.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+5.  On the example scheme we can see, that *Baboon* sends out the following data:
+    `Name`, `Surname`, `Address`, `Phone Number` and `Email`. Component *Toucan* doesn’t
+    need `Email`. Also, it has `Name and Surname` together in one field. Data mapper
+    makes the necessary changes into Baboon output, so that the data fits *Toucan*
+    input standard.
+
+![](/assets/img/getting-started/what-is-a-sample/Screenshot_1.png)
+
+**NOTE:** Not all components work in by the same process. Some of them, for example service components, have their own setup behavior.
```
Comment (drobiazko) [inline _getting-started/data-sample-overview.md]: Not sure what exactly you want to tell the reader here.

File: _getting-started/data-sample-overview.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+## Exceptions
```
Comment (drobiazko) [inline _getting-started/data-sample-overview.md]: The entire section is not clear to me. I think you should be more concrete here or remove it.

File: _getting-started/data-sample-overview.md (hunk 2 above)
Comment (honeybadgeralex) [reply _getting-started/data-sample-overview.md]: done

File: _getting-started/data-sample-overview.md (hunk 3 above)
Comment (honeybadgeralex) [reply _getting-started/data-sample-overview.md]: done

**Comment by honeybadgeralex** [comment]
Decided to kill a a section
